pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -31,6 +32,7 @@ import (
 	"k8s.io/kubernetes/pkg/api/v1"
 	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/lifecycle"
 	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+	hugepage "github.com/opencontainers/runc/libcontainer/configs"
```

Comment:
this alias seems a little weird -- how about this instead?

`libcontainerconfigs "github.com/opencontainers/runc/libcontainer/configs"`